packages/grafana-data/src/dataframe/StreamingDataFrame.ts

```diff
@@ -58,14 +43,39 @@ export class StreamingDataFrame implements DataFrame {
     return this.fields[0].values.length;
   }
 
-  update(msg: DataFrameJSON) {
-    if (msg.schema) {
-      // TODO, replace the existing fields
+  /**
+   * returns true if the operation was an append
```

Comment:
not in love with these semantics

Reply:
I just removed the return value.  If we need this distinction, we can add it later